VaticanBAV/et/BAVet175.xml

```diff
@@ -27,12 +30,313 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                            </msItem>
+                            
+                            <msItem xml:id="p1_i2.5">
+                              <locus from="22v" to="25v" />
+                              <title type="complete" ref="LIT2509Weddas#Friday" />
+                            </msItem>
+                            
+                            <msItem xml:id="p1_i2.6">
+                              <locus from="25v" to="28r" />
+                              <title type="complete" ref="LIT2509Weddas#Saturday" />
+                            </msItem>
+                            
+                            <msItem xml:id="p1_i2.7">
+                              <locus from="28r" to="30v" />
+                              <title type="complete" ref="LIT2509Weddas#Sunday" />
+                            </msItem>
+                          </msItem>
+                          
+                          <msItem xml:id="p1_i3">
+                            <locus from="31r" to="41r" />
+                            <title type="complete" ref="LIT1113Anqasa" />
+                            <textLang mainLang="gez"/>
+                            
+                            <msItem xml:id="p1_i3.1">
+                              <locus target="#41r" />
+                              <title type="complete" ref="LIT3108RepCh388" />
+                              <incipit xml:lang="gez">ይዌድስዋ፡ መላእክት፡ ለማርያም፡ በውስተ፡ ውሣጤ፡ መንጦላዕት፡ ወይብልዋ፡ በሐኪ፡ ማርያም<supplied reason="omitted">፡</supplied> <choice><sic>ሐዳስዩ፡</sic></choice> ጣዕዋ፨ ይቤላ፡ መልአክ፡ ለማርያም፡ ተወከፊዮ፡ ለቃል፡ ኀቤኪ፡ ይመጽእወበማኅፀነ፡ ዚአኪ፡ የኀድር፨</incipit>
+                            </msItem>
+                            
+                            <msItem xml:id="p1_i3.2">
+                              <locus from="41r" to="43v" />
+                              <title type="complete" ref="LIT6779Annun"/>
+                              
+                              <msItem xml:id="p1_i3.2.1">
+                                <locus from="41r" to="42v" />
+                                <title type="complete">Luke 1:26–38</title>
+                              </msItem>
+                              
+                              <msItem xml:id="p1_i3.2.2">
+                                <locus from="42v" to="43v" />
+                                <title type="complete" ref="LIT4272Salam" />
+                              </msItem>
+                            </msItem>
+                            
+                            <msItem xml:id="p1_i3.4">
+                              <locus from="43v" to="44r" />
+                              <title type="incomplete" ref="LIT1834Mahlet">Stanzas from Māḫleta ṣǝge</title>
+                              <incipit xml:lang="gez">
+                                <note>The incipit given by the catalogue is constituted by the end of stanza 50 and beginning of stanza 65.</note>
```

Comment:
Why do we call it an incipit then? I propose as a note: `<note>The end of stanza 50 (<foreign xml:lang="gez">በከመ፡ ሰገደ፡ ለወልድኪ፡ ዮሐንስ፡ በከርሥ።</foreign>) and the beginning of stanza 65 (<foreign xml:lang="gez">እንዘ፡ ተሐቅፊዮ፡ ለሕፃንኪ<supplied reason="omitted">፡</supplied> ጽጌ፡ ፀዓዳ፡ ወቀይህ።</foreign>) are given by the catalogue.</note>`

Reply:
I understood the catalogue to mean that the text it gives (identified by Augustine as the end of stanza 50 and beginning of stanza 65) is in fact the beginning of the text as it presents in this manuscript